addons/website_sale_autocomplete/models/res_config_settings.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    google_places_api_key = fields.Char('Google Places API Key', related='website_id.google_places_api_key', readonly=False)
```

Comment:
Why should we have 1 api key by website ? Shouldn't it be share across the websites with a config parameter like:
```suggestion
    google_places_api_key = fields.Char('Google Places API Key', config_parameter='website_sale_autocomplete.google_places_api_key')
```

Reply:
On the other hand, having the read access right on the website exposes the API key :(